tests/h/formatters/annotation_moderation_test.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from collections import namedtuple
+
+import pytest
+
+from h.formatters.annotation_moderation import AnnotationModerationFormatter
+
+FakeAnnotationResource = namedtuple('FakeAnnotationResource',
+                                    ['annotation', 'group'])
+
+
+class FakePermissionCheck(object):
+
+    def __init__(self):
+        self._permissions = {}
+
+    def add_permission(self, permission, context, granted):
+        self._permissions[(permission, context)] = granted
+
+    def __call__(self, permission, context):
+        return self._permissions[(permission, context)]
+
+
+class TestAnnotationModerationFormatter(object):
+    def test_preload_sets_flag_counts(self, db_session, flagged, unflagged, user):
+        formatter = AnnotationModerationFormatter(db_session,
+                                                  user,
+                                                  has_permission=None)
```

Comment:
`has_permission=None` is  presumably a convenience, in the sense that you just happen not to be calling any methods in this test that would explode because you haven't passed it in?